### ISSUE TITLE
Properly encode NameInfo

### ISSUE THREAD

**Issue by graup** [aergoio/herapy#68]
- Currently there is a useless nesting `name.name`
- Owner and destination are addresses, they should be encoded as base58check strings

**Comment by graup** [comment]
Current result:
```
{
name: {
name: "HunyoungPark"
},
owner: "A5aQo12vwwpMOyxycL3jwTY0m3K3PobDfKxFnw4rOypn",
destination: "A5aQo12vwwpMOyxycL3jwTY0m3K3PobDfKxFnw4rOypn"
}
```

Expected result:
```
{
name: "HunyoungPark",
owner: "AmPbHAMfJr42XM8jdPsaMDQkgJm5BJk4w1f35Nz82daiVwYMbK9U",
destination: "AmPbHAMfJr42XM8jdPsaMDQkgJm5BJk4w1f35Nz82daiVwYMbK9U"
}
```

**Comment by hanlsin** [comment]
It's because it is rendering all entities from grpc directly.

I'm working to support to create/update a name simultaneously.

It'll be included in v1.2.6.

**Comment by hanlsin** [comment]
It's included in v1.2.6.
